extensions/resteasy-classic/rest-client/config/src/main/java/io/quarkus/restclient/config/RestClientsConfig.java

```diff
@@ -45,4 +45,21 @@
     @ConfigItem
     public Optional<String> multipartPostEncoderMode;
 
+    /**
+     * Level of logging for the client.
+     * <br/>
+     * WARNING: beware of logging sensitive data
+     * <br/>
+     * The possible values are:
+     * <ul>
+     * <li>{@code request-response} - enables logging request and responses, including redirect responses</li>
+     * <li>{@code all} - enables logging requests and responses and lower-level logging</li>
+     * <li>{@code none} - no additional logging</li>
+     * </ul>
+     *
+     * This property is applicable to reactive REST clients only.
+     */
+    @ConfigItem
```

Comment:
is "logLevel" the right term ? we use "log.level" for debug/info/error/tracing else where which is not the same ....this here is more about "detailLevel" ?

Reply:
maybe `quarkus.rest-client.logging.scope` ?

Change from `-` to `.` would also open a possibility to have properties further customizing logging